includes/view/AngelTypes_view.php

```diff
@@ -358,7 +358,10 @@ function AngelType_view(
                 $angeltype,
                 $shiftsFilterRenderer,
                 $shiftCalendarRenderer
-            )
+            ),
+            __('Dashboard') => [
```

Comment:
Why use a tab as link? I would expect a link to another part of the app to be in the top button row.
Please add the dashboard icon.

Reply:
Maybe add it to the date select, hmm